indico/util/caching.py

```diff
@@ -30,7 +30,8 @@ def make_hashable(obj):
         return frozenset((k, make_hashable(v)) for k, v in obj.iteritems())
     elif hasattr(obj, 'getId') and obj.getId.__self__ is not None:
         # getId of AvatarUserWrapper would access a cached property, we can't have that here
-        id_ = obj.getId() if obj.__class__.__name__ != 'AvatarUserWrapper' else obj.id
+        # don't call Conference.getId() so we get the getId() access warning for other places
+        id_ = obj.getId() if obj.__class__.__name__ not in ('AvatarUserWrapper', 'Conference') else obj.id
```

Comment:
But. Where are we still using `Conference` objects? With no ZODB code left, where are they being stored/generated? I can find some 4 or 5 occurrences of `ConferenceHolder().getById(...)`. Maybe it's time to get rid of them?
(not suggesting that we do that in this PR, though)